### PR TITLE
COMMERCE-11277 Enable Request a Quote - Part 2 (Submission)

### PR DESCRIPTION
**Pull request by baharturk**
resent from #3695 with functional tests.

Story:
https://issues.liferay.com/browse/COMMERCE-11277

### CODE REVIEW

**Comment by baharturk** [comment]
ci:test:sf

**Comment by baharturk** [comment]
ci:test:sf

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceSalesAgent.testcase [hunk 1]
```diff
@@ -324,6 +394,69 @@ definition {
...
+				locator1 = "CommerceOrders#PLACED_ORDERS_DETAILS_STATUS");
+
+			Click.javaScriptClick(locator1 = "CommerceFrontStore#ORDER_PAGES_NOTES_BUTTON");
+
+			AssertTextNotPresent(
+				locator1 = "CommerceAccelerators#PORTLET_CONTENT",
+				value1 = "Edit");
+		}
+	}
+
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceSalesAgent.testcase:459]: Can this test and the one above be merged into one test?

**Review by lorenzolapescara — CHANGES_REQUESTED** [review]

**Comment by baharturk** [comment]
ci:test:commerce-functional

**Comment by brianikim** [comment]
ci:test:relevant

**Comment by brianikim** [comment]
@baharturk ,

The PR needs to be updated:

1. Your functional test is failing
2. Baseline changes have to be updated. Use my updated branch here: https://github.com/liferay/liferay-portal/compare/master...brianikim:liferay-portal:COMMERCE-11277

Sincerely,
Brian

**Comment by baharturk** [comment]
ci:test:sf

**Comment by baharturk** [comment]
ci:test:sf

**Comment by baharturk** [comment]
ci:test:relevant

**Comment by ling-alan-huang** [comment]
SF issue is not related to this PR, I fixed it in https://github.com/ling-alan-huang/liferay-portal/pull/1865

**Comment by victoroscuro** [comment]
ci:test:relevant

**Comment by victoroscuro** [comment]
ci:test:commerce-functional

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceSalesAgent.testcase [hunk 2]
```diff
@@ -211,6 +211,84 @@ definition {
 		}
 	}
 
+	@description = "COMMERCE-11277. As a Sales Agent, I can create a Quote Request in the Open Order's Details Page by entering a personal message."
+	@priority = 5
+	test CanCreateQuoteRequestAsSalesAgent {
+		property custom.properties = "feature.flag.COMMERCE-11028=true";
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceSalesAgent.testcase:217]: @baharturk this overrides the custom.properties set at definition level, and it is causing the test to fail on CI

Comment (baharturk) [reply modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceSalesAgent.testcase:217]: Thanks, I'll move it out of the test when I fix it.

**Comment by victoroscuro** [comment]
ci:forward:force

**Pull request closed without merging**